beeline/middleware/awslambda/test_awslambda.py

```diff
@@ -191,6 +198,30 @@ def handler(event, context):
                 'app.function_name': 'fn',
                 'app.function_version': '1.1.1',
                 'app.request_id': '12345',
-                'app.event': ANY,
+                'app.event': ANY, # 'app.event' is included by default
                 'meta.cold_start': ANY,
                 'name': 'handler'}, ANY)
+            m_add_context_field.assert_called_once_with('app.response', 1)
+
+    def test_can_omit_input_output(self):
+        ''' ensure input and output event fields can be omitted '''
+        with patch('beeline.propagate_and_start_trace') as m_propagate, \
+                patch('beeline.add_context_field') as m_add_context_field, \
+                patch('beeline.middleware.awslambda.beeline._GBL'), \
+                patch('beeline.middleware.awslambda.COLD_START') as m_cold_start:
+            m_event = Mock()
+            m_context = Mock(function_name='fn', function_version="1.1.1",
+                             aws_request_id='12345')
+
+            @awslambda.beeline_wrapper(record_input=False)
```

Comment:
Should this also set `record_output` to **False**? The assert below verifies `app.response` is not in the response object.

Reply:
Yeah, it should have... ended up just making separate tests for input/output in 593fe92b1bf5722d4daef5e59c6e1079a126dbb4